postman/schemas/transfer.yml

```diff
@@ -2705,6 +2747,13 @@ components:
           type: array
           items:
             type: string
+    transferProgressResponse:
+      type: object
+      properties:
+        transferId:
```

Comment:
I'm assuming transfer ID will be null until the transfer is complete? If so this should be described in the description for this property.